src/websocket.js

```diff
@@ -1,22 +1,21 @@
-import WebSocket from 'ws'
-import zip from 'lodash.zipobject'
-
 import httpMethods from 'http'
+import openWebSocket from 'open-websocket'
+import zip from 'lodash.zipobject'
```

Comment:
Would you mind putting vendors dependencies at the top and separated by a newline to our sources?